### DISCUSSION TITLE
Selection Visualizer - Griefdefender Support

### DISCUSSION THREAD

**Discussion by Parthurax** [Zrips/CMI#5126] [Enhancement]
Would be nice if SV can support **Griefdefender** just like Griefprevention.

**Comment by TWME-TW** [comment]
Hope to support but it's been a year

**Comment by mrfloris** [comment]
I think a while ago I asked about this, and I believe the unofficial response was that there's already griefprevention, and no interest currently to add griefdefender. Of course, this was a somewhat unofficial response, and maybe since time has passed this can be added now.
